github/ci/prow/files/jobs/kubevirt/kubevirt-presubmits.yaml

```diff
@@ -0,0 +1,234 @@
+presubmits:
```

Comment:
Since most of the configurations here are the same except for `name` and `image`, maybe YAML anchors could make it more readable? WDYT?

Reply:
Hm, it would always only save a minimal amount. To be honest I would right now prefer to simply see everything. We already use presets via the labels to avoid re-typing infra-related things. The rest is very similar but savings would be very similar and I find it this way more readable.